applications/nrf_desktop/src/modules/Kconfig.watchdog

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Watchdog settings"
+
+config DESKTOP_WATCHDOG_ENABLE
+	bool "Enable watchdog support"
+	depends on WATCHDOG
+	help
+		Enable watchdog to reset the device in case of problems.
+
+if DESKTOP_WATCHDOG_ENABLE
+
+config DESKTOP_WATCHDOG_TIMEOUT
+	int "Watchdog timeout value [s]"
+	default 16
+	range 16 120
```

Comment:
In my opinion constraint put on configuration should be done in config files (until there are strong reasons to do it in code or some where else). I susges it to leave it there. Values in range may be changes (lowe bound) when LOG_PANIC is change to some flush implementation, and upper bound if required may be changed right now.

Reply:
ok

Reply:
Why is the watchdog minimum value so high? My first thinking would be that we should be able to detect a locked up device faster than 16 seconds

Reply:
It is set to 16 second because of logging during power down. In such state LOG_PANIC is used, that blocks work queue and there is no way to feed watchdog. If the timeout is lower, then watchdow triggers board reset. We have to change implementation of logging during power down, if that is much more faster then the time may be changed to lower value.

Reply:
Let's switch watchdog timeout units to `ms`  (milliseconds), set the default to 500 (ms) and disable watchdog in debug builds (where it is not really needed).